doc/content/getting-started/programming-software/st-examples/mounting/_index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "USB Mounting"
+description: ""
+weight:
+distributions: null
+---
+
+A ST-LINK can work similarly to a USB drive, where you can simply place data on from your computer. When you place a binary executable, it will program it onto the {{% gn %}} automatically. This technique is good for people that do not want to install any extra software.  
+
+<!--more-->
+
+{{< note >}} This is not possible in early ST-LINK versions, it was introduced in ST-LINK/V3SET. {{</ note >}}
```

Comment:
I think they added support for V2 Drag and Drop recently, please check the [release notes ](https://www.st.com/content/ccc/resource/technical/document/release_note/98/de/c7/1b/08/82/44/38/DM00107009.pdf/files/DM00107009.pdf/jcr:content/translations/en.DM00107009.pdf)

Reply:
I'll generalise it a bit more then, in case of new releases and/or firmware upgrades